skbio/diversity/beta/_base.py

```diff
@@ -66,18 +84,18 @@ def pw_distances(counts, ids=None, metric="braycurtis"):
 
 @deprecated(as_of="0.4.0", until="0.4.1",
             reason=pw_distances_from_table_deprecation_reason)
-def pw_distances_from_table(table, metric="braycurtis"):
+def pw_distances_from_table(metric, table):
```

Comment:
Revert the API change to this function, since it's deprecated (no reason to modify the API, since it's going away).